crates/bitwarden-vault/src/cipher/cipher_client/admin/get.rs

```diff
@@ -0,0 +1,70 @@
+use bitwarden_api_api::models::CipherMiniDetailsResponseModelListResponseModel;
+use bitwarden_core::{ApiError, OrganizationId, key_management::KeyIds};
+use bitwarden_crypto::{CryptoError, KeyStore};
+use bitwarden_error::bitwarden_error;
+use thiserror::Error;
+
+use crate::{
+    VaultParseError,
+    cipher::cipher::{DecryptCipherListResult, PartialCipher},
+    cipher_client::admin::CipherAdminClient,
+};
+
+#[allow(missing_docs)]
+#[bitwarden_error(flat)]
+#[derive(Debug, Error)]
+pub enum GetOrganizationCiphersError {
```

Comment:
❓ Should this error type also have a reference to "Admin" somewhere in its name like all the other errors for the Admin client?